config/base.webpack.plugins.js

```diff
@@ -23,23 +23,23 @@ plugins.push(WriteFileWebpackPlugin);
  * @type {var}
  */
 const HtmlWebpackPlugin = new (require('html-webpack-plugin'))({
-    title: 'My App',
+    title: 'Home',
     filename: 'index.html',
     chunks: [ 'App', 'vendor' ],
     template: path.resolve(__dirname, '../src/index.html')
 });
 plugins.push(HtmlWebpackPlugin);
 
 const LogoutPage = new (require('html-webpack-plugin'))({
-    title: 'Logout',
+    title: 'Logged out',
```

Comment:
How about `RHCS | Logged out`

Reply:
As per above, we're not `RHCS`. We've also decided _not_ to use the pipes in the title as of https://github.com/RedHatInsights/landing-page-frontend/pull/34

This is tracked in RHCLOUD-1417

> 404 should say "Not found"
> Logout should say "Logged out"